iocore/net/quic/test/test_QUICFrame.cc

```diff
@@ -27,8 +27,18 @@
 #include "quic/QUICFrame.h"
 #include "quic/QUICStream.h"
 
+extern const ink_freelist_ops *freelist_global_ops;
+extern const ink_freelist_ops *freelist_class_ops;
+
 TEST_CASE("QUICFrame Type", "[quic]")
 {
+  // EThread *thread = new EThread();
+  // thread->set_specific();
+  init_buffer_allocators(0);
```

Comment:
Can we move these into main.cc or event_processor_main.cc?

Reply:
Will do.